KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutPresenter.swift

```diff
@@ -196,18 +196,22 @@ final class KarhooCheckoutPresenter: CheckoutPresenter {
     }
 
     // MARK: - Book
-    func bookTripPressed() {
-        guard areTermsAndConditionsAccepted else {
-            self.view?.showTermsConditionsRequiredError()
+    func startBooking() {
```

Comment:
Seeing that the booking process has already started for the user (they went through address selection, quote selection, etc.), this would be completing the flow, not starting it. Please rename the method to `completeBooking` or `completeBookingFlow`. Something along these lines.

Reply:
Done, renamed to completeBookingFlow